wallets/react-wallet-v2/src/utils/HederaWalletUtil.ts

```diff
@@ -0,0 +1,31 @@
+import { HederaWallet } from '@/lib/HederaLib'
+
+export let hederaAddresses: string[]
+export let hederaWallet: HederaWallet
+
+export async function createOrRestoreHederaWallet() {
+  try {
+    const accountId = process.env.NEXT_PUBLIC_HEDERA_ACCOUNT_ID
+    const privateKey = process.env.HEDERA_PRIVATE_KEY
```

Comment:
let's use local storage like these other projects

it looks like a few other projects use `local_storage` to get private keys for these demo apps, here's a few examples from https://github.com/WalletConnect/web-examples/tree/main/wallets/react-wallet-v2/src/utils 

https://github.com/WalletConnect/web-examples/blob/main/wallets/react-wallet-v2/src/utils/EIP155WalletUtil.ts
https://github.com/WalletConnect/web-examples/blob/main/wallets/react-wallet-v2/src/utils/SolanaWalletUtil.ts
https://github.com/WalletConnect/web-examples/blob/main/wallets/react-wallet-v2/src/utils/PolkadotWalletUtil.ts